src/screens/onboarding/components/step-content/experience-content.tsx

```diff
@@ -1,20 +1,42 @@
-import React from 'react';
+import React, { useCallback } from 'react';
 import { ScrollView, View } from 'react-native';
 
 import { HeadingLevel, OnboardingRoute } from 'src/common/enums';
-import { Heading, AddButton } from 'src/components';
-import { useAppNavigation } from 'src/hooks';
+import { ICareer } from 'src/common/types';
+import { Heading, AddButton, CareerCard } from 'src/components';
+import { useAppDispatch, useAppNavigation, useAppSelector } from 'src/hooks';
+import { experienceActions } from 'src/store/experience';
 import useStyles from './styles';
 
 const ExperienceContent: React.FC = () => {
   const styles = useStyles();
-
   const navigation = useAppNavigation();
+  const dispatch = useAppDispatch();
+
+  const { careerExperience } = useAppSelector(state => state.experience);
 
   const handleAddExperience = () => {
-    navigation.navigate(OnboardingRoute.ADD_EXPERIENCE);
+    navigation.navigate(OnboardingRoute.ADD_EXPERIENCE, {
+      career: undefined,
+    });
```

Comment:
Why do we need to change it to use undefined?

Reply:
I've added strongly types navigation and if I didn't pass any params, I get an error and I can fix it only by passing some params or just ` {} ` instead of ` {
      career: undefined,
    });`